### PR TITLE
NEPT-24: Poetry MOCK doesn't support some HTML tags.

### PR DESCRIPTION
**Pull request by dxvargas**

### CODE REVIEW

File: profiles/common/modules/custom/tmgmt_poetry/tmgmt_poetry_mock/src/Mock/PoetryMock.php [hunk 1]
```diff
@@ -294,15 +296,22 @@ public static function getDataFromRequest($message) {
    */
   private static function translateRequestContent($content, $language) {
     $decoded_content = base64_decode($content);
-    $xml_content = simplexml_load_string($decoded_content);
+    $dom = new \DOMDocument();
```
Comment (Fefaine) [inline profiles/common/modules/custom/tmgmt_poetry/tmgmt_poetry_mock/src/Mock/PoetryMock.php]: As i already expressed in ticket NEPT-71 the mock should not mimick the functions in tmgmt_poetry but use them.
I think this ticket is a good opportunity to fix this.

_tmgmt_poetry_replace_job_in_content

**Comment by ademarco** [comment]
👍

**Comment by ademarco** [comment]
I've given my +1 since this seems to fix the problem as agreed on the issue but @Fefaine should approve changes she requested in order to unblock this PR.

**Comment by Fefaine** [comment]
:+1:

**Comment by tatarbj** [comment]
:+1:

**Pull request merged**